iceoryx_examples/waitset_in_c/ice_c_waitset_grouping.c

```diff
@@ -71,7 +74,9 @@ int main()
     {
         subscriber[i] = iox_sub_init(&(subscriberStorage[i]), "Radar", "FrontLeft", "Counter", &options);
     }
+    //! [create subscriber]
 
+    //! [attach subscriber]
     const uint64_t FIRST_GROUP_ID = 123U;
     const uint64_t SECOND_GROUP_ID = 456U;
 
```

Comment:
next line:
attach the first two subscriber**s** to waitset with a triggerid of FIRST_GROUP_ID